setup.py

```diff
@@ -743,6 +743,7 @@ def is_package_excluded(package: str, exclusion_list: List[str]):
     'python-nvd3~=0.15.0',
     'python-slugify>=3.0.0,<5.0',
     'requests>=2.20.0, <3',
+    'rich>=9.2.0',
```

Comment:
We might need to add rich to https://github.com/apache/airflow/blob/master/LICENSE as it is MIT license

cc @ashb

Reply:
Good point!

Reply:
Added 👍